packages/containers/psammead-timestamp-container/README.md

```diff
@@ -22,8 +26,10 @@
 | prefix | string | No | `null` | `Updated` |
 | suffix | string | No | `null` | `This is a suffix` |
 | script | object | Yes | N/A | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, } |
+| locale | string | no | `null` | `en` |
```

Comment:
Can you add text somewhere that this is specifically the `moment.js` locale please